scripts/rebalance-queue-masters

```diff
@@ -74,11 +74,40 @@ set_temp_queue_policies()
     local updated_master=''
     local policy_set_master=''
     local -i count=0
-    local -i master_idx=0
 
     IFS=$'\n'
-    for queue_and_pid in $(rabbitmqctl -q -p "$vhost" list_queues name pid | grep -E "$queue_regex")
-    do
+    while true; do
+        # List of nodes
+        node_list=$(rabbitmqctl eval 'rabbit_nodes:all_running().' | sed -e "s/[][']//g" | tr ',' '\n')
```

Comment:
Does this need to be calculated on each iteration?

Reply:
Yes, in case of node shutdown in the middle of the rebalance operation

Reply:
If that happens, you've got problems :smile:

Reply:
But it's better than trying to rebalance queues masters to an unavailable node!! XD